UPGRADES.md

```diff
@@ -0,0 +1,94 @@
+# Upgrading an Existing Deployment
+
+## Overview
+
+This page shows you how to upgrade the release of a MATLAB Production Server:tm: deployment that is hosted on Amazon Web Services (AWS) and deployed using the reference architectures. 
+
+
+## Prerequisites 
+
+- An existing deployment of MATLAB Production Server, as deployed by the reference architecture. 
+- A backup of all applications uploaded to your current MATLAB Production Server deployment. You can create this by copying the contents of the S3 bucket containing your application code archives (``.ctf`` files) to a secure location. 
+- (Optional) An existing VPC with the required subnets and ports to deploy MATLAB Production Server to it. For more information on these requirements, navigate to the [reference architecture documentation](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws) for the MATLAB release of your existing deployment. Required subnets and ports are described in "Step 3. Configure Existing VPC" for R2023a and later, or "How do I use an existing VPC to deploy MATLAB Production Server" in the FAQ section for earlier releases. If you already have an existing deployment in this VPC, these requirements should already be met.
+
+
+## Considerations
+
+### URL Compatibility
+Your client applications might access MATLAB Production Server applications using the default MATLAB Execution Endpoint URL ```MATLABProductionServerFunctionExecutionURL``` provided in the **Outputs** tab of the root stack. If they do, note that this URL is different for your new deployment.  
+
+You can avoid this issue in future upgrades by setting a custom DNS name for the MATLAB Execution Endpoint load balancer ```MATLABProductionServerLoadBalancer```. See [Configure a custom domain name for your Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/using-domain-names-with-elb.html) for instructions.
```

Comment:
Since we are linking out to AWS doc, MathWorks doc center, and GitHub, it may be helpful to orient the reader. Editorial review should cover this.